stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -92,6 +92,18 @@ spec:
             - containerPort: {{ .Values.Master.JMXPort }}
               name: jmx
             {{- end }}
+{{- if .Values.Master.HealthProbes }}
+          livenessProbe:
+            httpGet:
+              path: /login
+              port: http
+            initialDelaySeconds: {{ .Values.Master.HealthProbesTimeout | default 120 }}
```

Comment:
please change this to `{{ .Values.Master.HealthProbesTimeout }}` and add the default value in the values.yaml file